notify/sms/sms.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sms
+
+import (
+	"errors"
+	"github.com/megaease/easeprobe/global"
+	"github.com/megaease/easeprobe/notify/sms/conf"
+	"github.com/megaease/easeprobe/notify/sms/nexmo"
+	"github.com/megaease/easeprobe/notify/sms/twilio"
+	"github.com/megaease/easeprobe/notify/sms/yunpian"
+	log "github.com/sirupsen/logrus"
+)
+
+//NotifyConfig implements the structure of Sms
+type NotifyConfig struct {
+	//Embed structure
+	conf.Options `yaml:",inline"`
+
+	Provider conf.Provider `yaml:"-"`
+}
+
+// Kind return the probe kind
+func (c *NotifyConfig) Kind() string {
+	return c.MyKind
+}
+
+// Config Sms Config Object
+func (c *NotifyConfig) Config(gConf global.NotifySettings) error {
+	c.DefaultNotify.Config(gConf)
+	c.configSmsDriver()
+	c.SendFunc = c.DoNotify
+
+	log.Debugf("[%s] configuration: %+v, %+v", c.NotifyStat, c, c)
```

Comment:
This log.Debugf() seems wrong...

Reply:
Let me see, and I'm working with the SLASummary, wait a minute

Reply:
> This log.Debugf() seems wrong...

try again